projects/aca-content/src/lib/dialogs/folder-details/folder-information.component.spec.ts

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { FolderInformationComponent } from './folder-information.component';
+import { DIALOG_COMPONENT_DATA, RedirectAuthService } from '@alfresco/adf-core';
+import { ContentService, NodesApiService } from '@alfresco/adf-content-services';
+import { By } from '@angular/platform-browser';
+import { EMPTY, of, Subject } from 'rxjs';
+import { LibTestingModule } from '@alfresco/aca-shared';
+
+describe('FolderInformationComponent', () => {
+  let fixture: ComponentFixture<FolderInformationComponent>;
+  let nodeService: NodesApiService;
+  let initiateFolderSizeCalculationSpy: jasmine.Spy;
+  let getFolderSizeInfoSpy: jasmine.Spy;
+  const mockSub = new Subject<{ entry: { jobId: string } }>();
```

Comment:
Instead defining new type please use just JobIdBodyEntry inside <>

Reply:
Updated to use JobIdBodyEntry